### DISCUSSION TITLE
Zoom Lecture Recording

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#227] [Archive] [answered]
Hi @soniamitchell @richardreeve,

When will the zoom recording be available on moodle? I want to be able to recap yesterdays lesson before tomorrows.

Thanks,

Sarah

**Comment by richardreeve** [comment, marked as answer]
Apologies for the delay in getting back to you on this, but Zoom took over a day to do the transcription that day - everything is on Moodle now.
